<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-D48UAL4I" failures="4" tests="8" name="Personal Loan" time="0.131" errors="0" timestamp="2025-05-10T20:04:51 IST">
  <testcase classname="firstPackage.Firstclass" name="secondTest" time="0.01"/>
  <testcase classname="firstPackage.Secondclass" name="test1" time="0.001"/>
  <testcase classname="firstPackage.HomeLoan" name="ApiLoginHomeLoan" time="0.002"/>
  <testcase classname="firstPackage.HomeLoan" name="mobileLoginHomeLoan" time="0.003"/>
  <testcase classname="firstPackage.HomeLoan" name="webLoginHomeLoan" time="0.013">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at firstPackage.HomeLoan.webLoginHomeLoan(HomeLoan.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- webLoginHomeLoan -->
  <testcase classname="firstPackage.HomeLoan" name="webLoginHomeLoan" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at firstPackage.HomeLoan.webLoginHomeLoan(HomeLoan.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- webLoginHomeLoan -->
  <testcase classname="firstPackage.HomeLoan" name="webLoginHomeLoan" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at firstPackage.HomeLoan.webLoginHomeLoan(HomeLoan.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- webLoginHomeLoan -->
  <testcase classname="firstPackage.HomeLoan" name="webLoginHomeLoan" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at firstPackage.HomeLoan.webLoginHomeLoan(HomeLoan.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- webLoginHomeLoan -->
  <testcase classname="firstPackage.Firstclass" name="firstTest" time="0.0">
    <ignored/>
  </testcase> <!-- firstTest -->
</testsuite> <!-- Personal Loan -->
